### PR TITLE
matching by category and russian template name

### PR DESCRIPTION
**Pull request by rybmat**
matching category that contains "infobox" and Инфобокс as template name

### CODE REVIEW

File: workers/template-classification-worker/src/main/java/com/wikia/workers/templateclassificationworker/matchers/CustomInfoboxMatcher.java [hunk 1]
```diff
@@ -25,7 +26,11 @@ private Boolean matchByClass(Document document) {
   }
 
   private Boolean matchByTemplateName(Document document) {
-    return document.title().toLowerCase().contains("infobox");
+    return document.title().toLowerCase().contains("infobox")
+        || document.title().toLowerCase().contains("Инфобокс".toLowerCase());
   }
 
+  private Boolean matchByCategory(Document document) {
+    return !document.select(("*[rel*=\"mw:PageProp/Category\"][href~=(?i)infobox]")).isEmpty();
```
Comment (WikiaTech) [inline workers/template-classification-worker/src/main/java/com/wikia/workers/templateclassificationworker/matchers/CustomInfoboxMatcher.java]: MAJOR Remove those useless parentheses. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: workers/template-classification-worker/src/main/java/com/wikia/workers/templateclassificationworker/matchers/CustomInfoboxMatcher.java [hunk 2]
```diff
@@ -25,7 +26,11 @@ private Boolean matchByClass(Document document) {
   }
 
   private Boolean matchByTemplateName(Document document) {
-    return document.title().toLowerCase().contains("infobox");
+    return document.title().toLowerCase().contains("infobox")
+        || document.title().toLowerCase().contains("Инфобокс".toLowerCase());
```
Comment (SebastianMarzjan) [inline workers/template-classification-worker/src/main/java/com/wikia/workers/templateclassificationworker/matchers/CustomInfoboxMatcher.java]: can we just pust "инфобокс" without conversion every time? :D

**Comment by SebastianMarzjan** [comment]
One minor comment, rest LG

**Pull request merged**